sdk/src/Core/Amazon.Runtime/Internal/Util/StringUtils.cs

```diff
@@ -241,8 +241,15 @@ public static string FromList(List<Enum> values)
         /// </summary>
         /// <param name="values">List of T</param>
         /// <returns>Header value representing the list of T</returns>
-        public static string FromValueTypeList<T>(List<T> values)  where T : struct
-        {
+        public static string FromList<T>(List<T> values)
```

Comment:
Are you sure this compiles changing it from `FromValueTypeList`? It shouldn't because it would conflict with the other FromList.

Also if this one is changes the `public static string FromValueTypeList<T>(IEnumerable<T> values) where T : struct` should also change.

Reply:
hmm this is odd because it builds in the main branch with all of our changes but not by itself. Investigating